waspc/cli/Command/Db/Migrate.hs

```diff
@@ -86,3 +94,39 @@ applyAvailableMigrationsAndGenerateClient genProjectRootDir = do
                                genClientError
         Right () -> waspSaysC "Done."
 
+
+data MigrationDirCopyDirection = CopyMigDirUp | CopyMigDirDown deriving (Eq)
+
+-- | Copy migrations directory between Wasp source and the generated project.
+copyDbMigrationsDir
+    :: MigrationDirCopyDirection -- ^ Copy direction (source -> gen or gen-> source)
+    -> Path Abs (Dir WaspProjectDir)
+    -> Path Abs (Dir ProjectRootDir)
+    -> IO (Maybe String)
```

Comment:
Mention it returns error if smth goes wrong. I thought it returns some value! Or you could use Either String (). Hm